### ISSUE TITLE
There's no decorators for the angular imports

### ISSUE THREAD

**Issue by darkv314** [formkit/auto-animate#210]
The directive and the module doesn't have the @Decorator or the @Module decorators and it's not possible to import them on standalone components

**Comment by Znegl** [comment]
I fixed it locally by providing my own decorator:
```typescript
import { Directive, ElementRef, inject } from '@angular/core';
import { AutoAnimateDirective as AutoAnimateDirective_ } from '@formkit/auto-animate/angular';

@Directive({
  selector: '[appAutoAnimate]',
})
export class AutoAnimateDirective extends AutoAnimateDirective_ {
  constructor() {
    const elementRef = inject(ElementRef);
    super(elementRef);
  }
}
```

**Comment by ajitzero** [comment]
Hi, this is a known issue #72, #57, #145 

TLDR; Use [`ng-auto-animate`](https://www.npmjs.com/package/ng-auto-animate). View [Source](https://github.com/ajitzero/ng-auto-animate/tree/main/libs/ng-auto-animate), licensed under MIT, same as this repo.

I need to release a new version with v21 support, but otherwise, you can easily copy the entire thing in your project.

> export class AutoAnimateDirective extends AutoAnimateDirective_ {

@Znegl This is pretty smart! I didn't consider extending it as-is.

If you'd like extra Angular-specific features like an `InjectionToken` for global settings, or SSR-safe isolation, consider using [`ng-auto-animate`](https://www.npmjs.com/package/ng-auto-animate).

**Comment by ajitzero** [comment]
@darkv314 @Znegl I've just released v1.0.0 with support for the latest Angular version.

Consider going through [the docs](https://github.com/ajitzero/ng-auto-animate/blob/main/libs/ng-auto-animate/README.md) if this is relevant for you.
